esmvalcore/preprocessor/_mask.py

```diff
@@ -285,9 +285,6 @@ def _get_geometries_from_shp(shapefilename):
         msg = f"Could not find any geometry in {shapefilename}"
         raise ValueError(msg)
 
-    # TODO: might need this for a later, more enhanced, version
-    # geometries = sorted(geometries, key=lambda x: x.area, reverse=True)
-
```

Comment:
why is it removing TODOs and NOTEs? This is not something I like the thing doing

Reply:
Since #389 ..

Reply:
let us remove it, not some machine that has no discrimination, and all it's doing is enforcing rules. In fact, I've started to have it up to here with Ruff rules in certain cases, there are a few times when things should be a lot more felxible but Ruff barks at you and I have to silence it with `--no-verify`, which is not good to do

Reply:
ah did you? I thought this was all the machine's edits. OK then, am happy - if you are removing these then pls do - just don't want the blithering thing to do it for me, silently, too

Reply:
![me typing](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNHM4d3Z6MDVmcDltZGt1OTg1eGVueHNjMHM3dXRjeDhweDhuamg2byZlcD12MV9naWZzX3NlYXJjaCZjdD1n/fQZX2aoRC1Tqw/giphy.gif)